application/pages/modplatform/flame/FlamePage.cpp

```diff
@@ -13,9 +13,19 @@ FlamePage::FlamePage(NewInstanceDialog* dialog, QWidget *parent)
     ui->setupUi(this);
     connect(ui->searchButton, &QPushButton::clicked, this, &FlamePage::triggerSearch);
     ui->searchEdit->installEventFilter(this);
-    model = new Flame::ListModel(this);
-    ui->packView->setModel(model);
+    listModel = new Flame::ListModel(this);
+    ui->packView->setModel(listModel);
+
+    ui->sortByBox->addItem(tr("Sort by featured"));
```

Comment:
Might be worth noting here that the index is a Curse implementation detail, and is important.

Reply:
added a note